### ISSUE TITLE
NTR: Tympanic membrane inflamed or bulging (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7813]
New term request Tympanic membrane inflamed or bulging (nadea3_190123193204): Erythematous, purulent, loss of bony landmarks, absent light reflex, TM; middle ear effusion  UMLS: 0578160

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000172
- label: Tympanic membrane bulging
- parents: [Abnormality of the tympanic membrane (HP:0040090)](https://hpo.jax.org/app/browse/term/HP:0040090)
- pmid: PMID:24134083
- def: The tympanic membrane protrudes (swells outwards). This feature may be observed with otoscopy.
- comment: See Figure 1 of PMID:24134083.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7813
- synonyms:
